user_journey/views/cohort_analysis/clients_last_seen.view.lkml

```diff
@@ -0,0 +1,71 @@
+view: clients_last_seen {
+  derived_table:{
+    sql:
+      SELECT
+        *
+      FROM
+        `moz-fx-data-shared-prod`.search.search_clients_last_seen AS clients_last_seen
+      WHERE
+        {% condition cohort_analysis.date %} CAST(DATE_SUB(clients_last_seen.submission_date, INTERVAL 56 DAY) AS TIMESTAMP) {% endcondition %}
+        OR clients_last_seen.submission_date = DATE_SUB(current_date, INTERVAL 2 DAY);;
+  }
+
+  dimension: submission_date {
+    type: date
+    sql: ${TABLE}.submission_date ;;
+    hidden: yes
+  }
+
+  dimension: client_id {
+    type: string
+    sql: ${TABLE}.client_id ;;
+    hidden: yes
+  }
+
+  dimension: sample_id {
+    type: string
+    sql: ${TABLE}.sample_id ;;
+    hidden: yes
+  }
+
+  dimension: day_diff_cohorts {
+    type: number
+    sql:  DATE_DIFF(${TABLE}.submission_date, ${cohort_analysis.submission_date}, DAY);;
+    hidden: yes
+  }
+
+  dimension: days_seen_bytes {
+    type: string
+    sql: ${TABLE}.days_seen_bytes ;;
+  }
+
+  dimension: days_seen_bits {
+    type: string
+    sql:
+      IF(${day_diff_cohorts} >= days_since_message.days_since_message,
```

Comment:
We only want to include a row if it has the full history. So if e.g. we're  looking at 8 `days_since_message`, but `search_clients_last_seen` is only 7 days in the future, we do not want to include that row. We need this because we're querying across multiple days of `events_daily`, such that some will have enough history and some won't.